python/src/pyarrow/adapters/pandas.cc

```diff
@@ -17,17 +17,18 @@
 
 // Functions for pandas conversion via NumPy
 
-#include <Python.h>
```

Comment:
I think this include was needed for older Numpies, e.g. 1.8 and 1.9. (We should still support them as those are the default ones you should build `manylinux1` packages against.) Will re-add (with an explanative comment) if this was a problematic change.